Scoop/Controller/Onboarding/ProfilePictureViewController.swift

```diff
@@ -63,17 +64,52 @@ class ProfilePictureViewController: OnboardingViewController {
         pictureImageView.addGestureRecognizer(tapGesture)
     }
     
-    private func setupTitleLabel() {
-        titleLabel.font = .systemFont(ofSize: 20, weight: .semibold)
-        titleLabel.text = "Hi Reade! Let's begin by adding a profile picture."
-        titleLabel.textColor = .black
-        titleLabel.numberOfLines = 0
-        titleLabel.textAlignment = .center
-        view.addSubview(titleLabel)
+    private func setupButtons() {
+        let buttonMultiplier = 0.09
+        let screenSize = UIScreen.main.bounds
+        let skipAction = UIAction { _ in
+            self.dismiss(animated: true)
+        }
+        
+        skipButton.setTitle("  Add photo later  ", for: .normal)
```

Comment:
Instead of adding spaces to the title string, you might want to setup `insets` instead.